genomic_data_service/region_indexer_elastic_search.py

```diff
@@ -117,6 +117,12 @@ def get_chrom_index_mapping(self, assembly='hg19'):
                     'effect_size': {
                         'type': 'string'
                     },
+                    'pwm_score': {
+                        'type': 'string'
+                    },
+                    'pwm_pvalue': {
```

Comment:
I wonder if these should be numeric fields not strings?  I guess they are all strings.
I also wonder if we should reuse p_value (I think that is for xQTLs) and value=score.
Sorry I didn't notice this before.